paper/paper.bib

```diff
@@ -0,0 +1,70 @@
+@software{PyYAML:2024,
+  author = {{YAML and Python communities}},
+  title = {PyYAML},
+  year = {2024},
+  month = {Aug},
+  version = {latest},
+  url = {https://github.com/yaml/pyyaml},
+}
+
+@software{Click:2025,
+    author       = {{The click development team}},
+    title        = {Command Line Interface Creation Kit},
+    month        = {May},
+    year         = {2025},
+    version      = {latest},
+    url          = {https://github.com/pallets/click}
+}
+
+@software{lxml:2025,
+    author = {{The lxml development team}},
```

Comment:
Behner et al.

Reply:
If there are authors we should name them, we unfortunately do not have the space to name them all that's why we use Behner et al., mentioning anybody is not only inconsistent but also poor if we know who the people are.